packages/block-library/src/index.js

```diff
@@ -3,7 +3,6 @@
  */
 import '@wordpress/core-data';
 import '@wordpress/block-editor';
-import '@wordpress/editor';
```

Comment:
I think we can't do that yet, because we still depend on `core/editor` store. (without checks for existence)

Reply:
See the description:

> This PR should not be merged yet because the widgets screen starts breaking because of 3 things:

:)

Reply:
haha :)

Reply:
> The reasonable blocks

That's a new thing to me. I prefer the non-reasonable blocks  :P

Reply:
This seems to be still unresolved. I'm worried this could affect the script loading order in some cases and break the UI since the package seems to be use in the reusable block.